src/Controller/MercadopagoController.php

```diff
@@ -108,7 +110,7 @@ public function oAuthCallbackAction(
         if (isset($res['error']) && !empty($res['error'])) {
             $this->addFlash(
                 'error',
-                $res['error_description']
+                $res['error']
```

Comment:
When I got some errors doing tests against Mercadopago API I saw that the field `error_description` was not present in the response. I think we should take `error_description` when it is present or `error` when it isn't but I don't know how to make that in PHP :)

Reply:
Like this:
```php
$res['error_description'] ?? $res['error']
```